django/db/models/sql/query.py

```diff
@@ -1762,6 +1761,10 @@ def clear_limits(self):
         """Clear any existing limits."""
         self.low_mark, self.high_mark = 0, None
 
+    @property
+    def has_limit(self):
```

Comment:
Should we name this `is_sliced` instead? In the case of `[10:]` which translates to `low_mark=10;high_mark=None` there's no `LIMIT` on the query. It also seems more in line with its current usages.

Reply:
Good point, I will amend this later today.

Reply:
Done.